### ISSUE TITLE
Incompatible with Python3 error on Google Cloud

### ISSUE THREAD

**Issue by jstolp** [bitnami/bitnami-docker-wordpress#199]
Click the "Deploy image To GCE button"

Receive error:

```
The template is incompatible with Python3. Please fix the following errors: Exception in software_status.py Traceback (most recent call last): return constructor[m](evaluation_context) File "software_status.py", line 188, in GenerateConfig File "software_status.py", line 161, in _Waiter File "software_status.py", line 131, in _WaiterDependsOn AttributeError: module 'types' has no attribute 'StringTypes' Resource: software_status.py Resource: config https://cloud.google.com/deployment-manager/docs/migrate-to-python3 
```

**Comment by marcosbc** [comment]
Hi @jstolp, could you clarify which "Deploy image to GCE" button you are referring to?

**Comment by beltran-rubo** [comment]
We just double checked the Bitnami WordPress from the Marketplace it is deployed successfully. https://console.cloud.google.com/marketplace/details/bitnami-launchpad/wordpress 

We got that message as well, it is a warning from the Google Console. Thanks for reporting it, we will work with Google Marketplace to solve it.

**Comment by jstolp** [comment]
Sure!

https://console.cloud.google.com/marketplace/details/click-to-deploy-images/wordpress?hl=nl

**Comment by jstolp** [comment]
Thanks in advance

> We just double checked the Bitnami WordPress from the Marketplace it is deployed successfully. https://console.cloud.google.com/marketplace/details/bitnami-launchpad/wordpress
> 
> We got that message as well, it is a warning from the Google Console. Thanks for reporting it, we will work with Google Marketplace to solve it.

Thanks in advance! :)

**Comment by iTerryAllen** [comment]
Am I still able to run the multi bitnami site even with this message from google cloud console:

"The template is incompatible with Python3. Please fix the following errors: Exception in software_status.py Traceback (most recent call last): return constructor[m](evaluation_context) File "software_status.py", line 188, in GenerateConfig File "software_status.py", line 161, in _Waiter File "software_status.py", line 131, in _WaiterDependsOn AttributeError: module 'types' has no attribute 'StringTypes' Resource: software_status.py Resource: config https://cloud.google.com/deployment-manager/docs/migrate-to-python3

**Comment by iTerryAllen** [comment]
Google Wordpress multisite is giving same error report.  It seems its something in the coding with python on google cloud.  I believe they are releasing updates. I'm stuck now. I need that multisite to work. Same error:

The template is incompatible with Python3. Please fix the following errors: Exception in software_status.py Traceback (most recent call last): return constructor[m](evaluation_context) File "software_status.py", line 188, in GenerateConfig File "software_status.py", line 161, in _Waiter File "software_status.py", line 131, in _WaiterDependsOn AttributeError: module 'types' has no attribute 'StringTypes' Resource: software_status.py Resource: config https://cloud.google.com/deployment-manager/docs/migrate-to-python3

**Comment by iTerryAllen** [comment]
How can we update python? Does anyone know of a code to make it easy?

Bitnami Forum
"Hello Bitnami,

You need to update you need to update your code to start using python 3. I was able to fix this by updating unicode to str type.

@limitlessv

let me know if you still need help."

**Comment by beltran-rubo** [comment]
Hi @iTerryAllen 

Unfortunately Bitnami does not have control over those Python scripts. We already contacted Google Marketplace team in order to fix them.

Based on our tests, this is only a warning for updating the scripts to Python 3. It does not affect the deployment and the image should be up & running. Did you find any other issue with WordPress Multisite?

**Comment by iTerryAllen** [comment]
Hi @beltran-rubo 

There was an issue an issue with one of the plugins i waas using after deploying the wordpress multisite container.  Upon installation of the "WooCommerce" plugin, the admin area along with entire site would lose its style sheet.  

I went to the main Bitnami website and creates a virtual machine with the multisite for google cloud and was able to deploy with no warning from google. But after installing the plugin from there everything seems fine except parts of the style sheet in admin were not functioning.

in Short, the issue was SSL redirections which I fixed from Cloudflare page rules.  

The Loadbalancers help a lot and site is working much faster now.  Thanks for your reply.

**Comment by beltran-rubo** [comment]
Glad to hear that. In that case the issue was not related to the Python3 warning from the Google Marketplace portal. We will close the case when the issue is solved and the warning disappears.

**Comment by andresbono** [comment]
Hi @jstolp, @iTerryAllen, the warning regarding Python 3 should not appear anymore.
